spec/acceptance/class_logstash_spec.rb

```diff
@@ -1,40 +1,41 @@
+# frozen_string_literal: true
+
 require 'spec_helper_acceptance'
 
 shared_examples 'a logstash installer' do
-  it "should install logstash version #{LS_VERSION}" do
+  it "installs logstash version #{LS_VERSION}" do
```

Comment:
They used `LS_VERSION` to define what version of logstash to install and test.
We should probably iterate over `6.8.x`, `7.17.x` for now. Found [these](https://www.elastic.co/support/eol) from this doc.

Reply:
there's a list of versions in https://www.elastic.co/guide/en/logstash/7.17/logstash-7-17-0.html . for be beginning, I only configured 7.17.6